### PR TITLE
fix tests on master

### PR DESCRIPTION
**Pull request by KristofferC**
Also fixes a bug when throwing error from the julia evaluate mode.

### CODE REVIEW

File: src/Debugger.jl [hunk 1]
```diff
@@ -51,6 +51,16 @@ function DebuggerFramework.locdesc(frame::JuliaStackFrame, specslottypes = false
     end
 end
 
+function sparam_syms(meth::Method)
```
Comment (KristofferC) [inline src/Debugger.jl]: @timholy is this a decent way to get the symbols for the parameters now that `sparam_syms` are gone from `Method`?

Comment (timholy) [reply src/Debugger.jl]: :+1: You've checked they come out in the right order? E.g., `AbstractArray{T,N} where T where N` doesn't swap `T` and `N`?

Comment (KristofferC) [reply src/Debugger.jl]: Yes, it seems ok:

```
> f(a::Int, b, c::T, d::T, e::F, g) where {T, F <: Integer} = a + b + c + d + e> f (generic function with 1 method)

> @enter f(1, 2.0, 3.0, 4.0, 5, 6)
In f(a, b, c, d, e, g) at REPL[4]:1
1 f(a::Int, b, c::T, d::T, e::F, g) where {T, F <: Integer} = a + b + c + d + e

About to run: (+)(1, 2.0, 3.0, 4.0, 5)
1|debug > fr
[1] f(a, b, c, d, e, g) at REPL[4]:1
  | a::Int64 = 1
  | b::Float64 = 2.0
  | c::Float64 = 3.0
  | d::Float64 = 4.0
  | e::Int64 = 5
  | g::Int64 = 6
  | T::DataType = Float64
  | F::DataType = Int64
1|debug > 
```

Comment (timholy) [reply src/Debugger.jl]: Great. If you want to add any more challenging signatures to the tests, there's a "zoo" in JuliaInterpreter's `toplevel_script.jl`. But I just checked
```julia
f(x::AbstractArray{V,N}) where {V<:Tuple{T,T},N} where T<:Real = 1
x = reshape(reinterpret(Tuple{Float64,Float64}, rand(2,3,3)), 3, 3)
@enter f(x)
```
and it was indeed fine.

Comment (KristofferC) [reply src/Debugger.jl]: Thanks. Overall, we need to add more tests to the for the output than good old `my_gcd`.

**Pull request merged**